flight/Modules/Actuator/actuator.c

```diff
@@ -340,12 +340,23 @@ static void actuatorTask(void* parameters)
 				continue;
 			}
 
-			if((mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_MOTOR) || (mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_SERVO))
+			if((mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_MOTOR) || (mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_SERVO)) {
 				status[ct] = ProcessMixer(ct, curve1, curve2, &mixerSettings, &desired, dT);
-			else
-				status[ct] = -1;
-
 
+				// If a motor type, additional work to be done
+				if(mixers[ct].type == MIXERSETTINGS_MIXER1TYPE_MOTOR) {
+					if (status[ct] > 0) {
+						// Apply curve fitting, mapping the input to the propeller output.
+						status[ct] = actuatorSettings.MotorInputOutputCurveFit[ACTUATORSETTINGS_MOTORINPUTOUTPUTCURVEFIT_A] *
```

Comment:
Yah.  I understand what you're saying.  And those are nice rational units to come into actuator.  And this makes a lot more sense when the controlled value is RPM.   When it's PWM (which basically all of our users are running) / some coarse idea of actuator effort on an ESC I'm not sure how true this holds.  Most people doing 3D are using (nearly) symmetric airfoils-- the main difference are prop texture and what side of the airstream the arm is on.     It's also problematic because then negative throttle needs a different curve for the 3D case, because you'll almost certainly not get as much negative thrust as positive.  And there's going to be something icky around 0 throttle no matter what-- both linearity from command to output but also strange time-domain effects.

Basically, I think the difference between A_reversed and just doing it one sided is something like: https://www.desmos.com/calculator/dcd8rumbqm

All in all, it's a difficult set of tradeoffs.

What we've been discussing on IRC for when we have smart ESCs (like perhaps ESC32 integration) is to keep something much like the current actuator effort on the output,  but to run inner angular acceleration and throttle loops based on ESC power or RPM feedback from an inverted mixer matrix.  The higher frequency components of that data could also be (weakly) blended into the rate estimate to further improve the bandwidth of the rate and attitude loops.

So it's an interesting question.  Obviously predistorting/linearize the command is helpful in either case.